index.bs

```diff
@@ -11173,11 +11135,14 @@ Here's how the above diagram could be constructed in JavaScript:
 <h2 id="DynamicLifetime">
 Dynamic Lifetime</h2>
 
-<h3 id="dynamic-lifetime-background">
+<h3 id="dynamic-lifetime-background" class="informative">
 Background</h3>
 
-<em>This section is non-normative. Please see <a href="#lifetime-AudioContext">AudioContext lifetime</a> and <a href="#lifetime-AudioNode">AudioNode lifetime</a> for normative
-requirements.</em>
+Note: The normative description of {{AudioContext}} and {{AudioNode}} lifetime characteristics is
+	available at <a href="#lifetime-AudioContext">AudioContext lifetime</a> and <a
```

Comment:
Maybe replace "is available at" with "is described by the AudioContext lifetime and AudioNode lifetime"? (With links you already have.)

Reply:
Fixed.